easylist_adult/adult_specific_hide.txt

```diff
@@ -198,7 +199,7 @@ vidgrab.net,xxvideo.us###player > #stop
 smut6.com###player-banner
 megatube.xxx###player-pop-layer
 ultrahorny.com###player12
-hentai2w.com,ultrahorny.com,xxxkingtube.com###playerOverlay
+hentai2w.com,ultrahorny.com,xxxkingtube.com,xxxity.com###playerOverlay
```

Comment:
```
whois xxxkingtube.com
   Domain Name: XXXKINGTUBE.COM
   Registry Domain ID: 1986192367_DOMAIN_COM-VRSN
   Registrar WHOIS Server: whois.godaddy.com
   Registrar URL: http://www.godaddy.com
   Updated Date: 2020-01-23T10:21:07Z
   Creation Date: 2015-12-08T09:22:37Z
   Registry Expiry Date: 2020-12-08T09:22:37Z
   Registrar: GoDaddy.com, LLC
   Registrar IANA ID: 146
   Registrar Abuse Contact Email: abuse@godaddy.com
   Registrar Abuse Contact Phone: 480-624-2505
   Domain Status: clientDeleteProhibited https://icann.org/epp#clientDeleteProhibited
   Domain Status: clientRenewProhibited https://icann.org/epp#clientRenewProhibited
   Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
   Domain Status: clientUpdateProhibited https://icann.org/epp#clientUpdateProhibited
   Name Server: NS1.PARKLOGIC.COM
   Name Server: NS2.PARKLOGIC.COM
```

You should move `xxxkingtube.com` to the spam list as it is hijacked by the cyber criminals at park(un)logic

Reply:
`xxxity.com` is actually also hijacled.... https://github.com/mypdns/matrix/issues/237